src/Metadata/Resource/Factory/OperationDefaultsTrait.php

```diff
@@ -208,7 +208,7 @@ private function getDefaultOperationName(HttpOperation $operation, string $resou
         return sprintf(
             '_api_%s_%s%s',
             $path ?: ($operation->getShortName() ?? $this->getDefaultShortname($resourceClass)),
-            strtolower($operation->getMethod() ?? HttpOperation::METHOD_GET),
```

Comment:
Not sure about this change, should it be rollbacked too?

Reply:
its fine as well an http operation has always a method